src/content/docs/en/pages/main-menu/reference/marketplace/marketplace-seller-guide.mdx

```diff
@@ -1,141 +1,213 @@
-| 3    | \$100,000 \- $999,999       | 10%                | 9%     | 8%     |
-| 4    | $1,000,000+                 | 9%                 | 7%     | 6%     |
+- For metered subscriptions and monthly Saving Plans, the value is the amount paid by the buyer each month.
+- For annual or multi-year subscriptions the value for the entire duration of the subscription is based on the amount paid upfront by the buyer at the time the buyer enters into the transaction.
 
->  The dollar value tiers in the table above are calculated as follows: (i) for metered subscriptions and monthly Saving Plans the value is the amount paid by the customer each month and (ii) for annual or multi-year subscriptions the value for the entire duration of the subscription is based on the amount paid upfront by the customer at the time the customer enters into the Transaction.
+Metered subscriptions, monthly Saving Plans, and 1-year Saving Plans will receive the “Monthly and Annual” Tiered Listing Fee until the beginning of the second year, at which point the Tiered Listing Fee will drop to the “Year 2” value (and so on to Year 3). Multi-year purchases that are either billed upfront or on a set schedule, will receive the Listing Fee applicable to the last year of the subscription for the entire transaction.
 
-Metered subscriptions, monthly Saving Plans and 1-year Saving Plans will receive the “Monthly and Annual” Tiered Listing Fee until the beginning of the second year, at which point the Tiered Listing Fee will drop to the “Year 2” value (and so on to Year 3). Multi-year purchases that are either billed up front, or on a set schedule, will receive the Listing Fee applicable to the last year of the subscription for the entire Transaction.
+For example:
 
-For example, if a customer is subscribed to a SaaS product, and pays \$19,000 per month for a metered subscription, the Listing Fee will be 13% for each monthly transaction (Tier 1, Monthly rate). In month 11, the customer increases their usage and is billed \$21,000, so the Listing Fee drops to 11% (Tier 2, Monthly rate). In month 13, the customer has maintained their usage and is billed ​\$21000, but because the subscription has entered year 2, the Listing Fee drops to 10% (Tier 2, Year 2 rate). In month 14, the customer decreases their usage to $19,000, and the Listing Fee increases to 12% (Tier 1, Year 2 rate).
+-   If a buyer is subscribed to a SaaS product and pays $19,000 per month for a metered subscription, the Listing Fee will be 13% for each monthly transaction (Tier 1, Monthly rate).
+-   In month 11, the customer increases their usage and is billed $21,000, so the Listing Fee drops to 11% (Tier 2, Monthly rate). In month 13, the customer has maintained their usage and is billed $21000, but because the subscription has entered year 2, the Listing Fee drops to 10% (Tier 2, Year 2 rate).
+-   In month 14, the customer decreases their usage to $19,000, and the Listing Fee increases to 12% (Tier 1, Year 2 rate).
```

Comment:
```suggestion
- If a buyer is subscribed to a SaaS product and pays $19,000 per month for a metered subscription, the Listing Fee will be 13% for each monthly transaction (Tier 1, Monthly rate).
- In month 11, the customer increases their usage and is billed $21,000, so the Listing Fee drops to 11% (Tier 2, Monthly rate). In month 13, the customer has maintained their usage and is billed $21000, but because the subscription has entered year 2, the Listing Fee drops to 10% (Tier 2, Year 2 rate).
- In month 14, the customer decreases their usage to $19,000, and the Listing Fee increases to 12% (Tier 1, Year 2 rate).
```